homeassistant/components/soma/config_flow.py

```diff
@@ -0,0 +1,65 @@
+"""Config flow for Soma."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import callback
+from homeassistant.const import (
+    CONF_HOST, CONF_PORT)
+from .const import DOMAIN, HOST, PORT
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_PORT = 3000
+
+
+@callback
+def register_flow_implementation(hass, host, port):
+    """Register a flow implementation."""
+    hass.data[DOMAIN][HOST] = host
+    hass.data[DOMAIN][PORT] = port
+
+
+@config_entries.HANDLERS.register('soma')
+class SomaFlowHandler(config_entries.ConfigFlow):
```

Comment:
Remove the decorator and pass the domain as domain keyword.
```suggestion
class SomaFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
```

Reply:
Fixed